### ISSUE TITLE
"O Haze" dataset evaluation

### ISSUE THREAD

**Issue by chen123hao** [jinyeying/FogRemoval#14]
Dear esteemed author, I would like to inquire about two matters:

It appears that pre-trained models for the "O Haze" dataset have not been provided. Could you confirm if all datasets were tested using the "NH Haze" pre-trained model you supplied?
The resolution of the dehazed images from the O Haze test set that you showcased is notably lower than that of the original dataset. Did you perform resizing on the original images during the testing phase?

**Comment by jinyeying** [comment]
> It appears that pre-trained models for the "O Haze" dataset have not been provided. Could you confirm if all datasets were tested using the "NH Haze" pre-trained model you supplied?

1. Hi, sorry for the late reply. For each dataset, we trained and tested different models, not one pre-trained model.

2. The O Haze test set, the resolution for 03.png is 768*512, for 07.png is 784*521, is this you mean resize?
